webapp/src/js/services/send-message.js

```diff
@@ -162,25 +163,25 @@ angular
       return task;
     };
 
-    return function(recipients, message) {
-      if (!_.isArray(recipients)) {
+    return (recipients, message) => {
+      if (!Array.isArray(recipients)) {
         recipients = [recipients];
       }
       return $q
         .all([UserSettings(), Settings(), formatRecipients(recipients)])
-        .then(function(results) {
-          var user = results[0];
-          var settings = results[1];
-          var explodedRecipients = results[2];
-          var doc = createMessageDoc(user);
-          doc.tasks = explodedRecipients.map(function(recipient) {
+        .then(([ user, settings, explodedRecipients ]) => {
+          const doc = createMessageDoc(user);
+          doc.tasks = explodedRecipients.map(recipient => {
             return createTask(settings, recipient, message, user);
           });
           self.setLastChangedDoc(doc);
           return doc;
         })
-        .then(function(doc) {
-          return DB().post(doc);
+        .then(doc => {
+          return $q.all([
+            DB().post(doc),
+            MarkRead([ doc ])
```

Comment:
Another change to improve e2e test stability (as well as make the UI better) - if you're creating a doc it is automatically marked as read so it only needs to be rendered once and the user is not bugged to read a message they created.